nemo/collections/asr/models/ctc_models.py

```diff
@@ -420,6 +423,61 @@ def _setup_transcribe_dataloader(self, config: Dict) -> 'torch.utils.data.DataLo
         temporary_datalayer = self._setup_dataloader_from_config(config=DictConfig(dl_config))
         return temporary_datalayer
 
+    def export(
```

Comment:
I believe it is going to be cleaner if we do not overload export() - but instead either implement symbolic() method that would only call encoder/decoder, or extract an Exportable superclass with forward() method that would do the same. I would prefer the latter because this way we would not have to 'cheat' by having exported graph that is not equivalent to forward().

Reply:
Actually, scratch that - since it's not module, but model, it does make sense to implement export() by fusion of exported ONNX graphs.  export() is good. We do need a unit test though.

Reply:
done. thanks!